tests/inference/test_example_query_openai_sdk.py

```diff
@@ -57,10 +64,15 @@ def script_with_args(api_base, model_name, streaming_response, max_new_tokens, t
 
     result_openai = subprocess.run(cmd_openai, capture_output=True, text=True)
 
+    # Ensure there are no errors in the OpenAI API query script execution
+    try:
```

Comment:
this is a test, you need to raise the exception rather than capture it.

Reply:
when something wrong, we also need to print serve stderr to check what is going on.